backend/src/main/java/com/google/sps/servlets/SearchMatchServlet.java

```diff
@@ -92,14 +97,59 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     participantDatastore.removeParticipant(match.getFirstParticipantUsername());
     participantDatastore.removeParticipant(match.getSecondParticipantUsername());
 
+    sendMatchResponse(response, match);
+  }
+
+  /** Return true if expired and should be removed, false if still valid */
+  private boolean checkExpired(Participant participant) {
+    long currentTimeMillis = System.currentTimeMillis();
+    return (participant.getEndTimeAvailable()
```

Comment:
This took me a bit to parse so a comment might be helpful to any future readers. To make sure I understand correctly, if I'm available until 2pm and want a meeting duration of 30 minutes, I'll be removed as expired 40 minutes before 2pm at 1:20pm (30 minutes + 10 minute padding)?